# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/video-rvfc/request-video-frame-callback-webrtc.https.html.ini
# Commit: 35d86243cb45
# Full Hash: 35d86243cb45f7e02dd6ec0eea04cd62c2a70196
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2024-09-16 09:14:26
# Regressor Bug: 1908245
# File Overlap Count: 1
# Description:
#   Bug 1908245 - Add support for missing requestVideoFrameCallback parameters. r=media-playback-reviewers,webidl,padenot,saschanaz
#   
#   This patch adds support for exposing the capture time, receive time,
#   processing duration and RTP timestamp parameters on the callback for
#   rVFC for WebRTC related elements.
# ==============================================================================

diff -r 7d143d92bf2a -r 35d86243cb45 testing/web-platform/meta/video-rvfc/request-video-frame-callback-webrtc.https.html.ini
--- a/testing/web-platform/meta/video-rvfc/request-video-frame-callback-webrtc.https.html.ini	Sun Sep 15 20:03:40 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-[request-video-frame-callback-webrtc.https.html]
-  [Test video.requestVideoFrameCallback() parameters for WebRTC applications.]
-    expected: FAIL
